### ISSUE TITLE
Feature: Auto-play setting for photosensitive pieces

### ISSUE THREAD

**Issue by 404src** [versumstudios/report-bugs#48]
I noticed that auto-play is disabled on the feed/profile for pieces that are labeled with the photosensitive seizure warning, which is a great design choice. I would love to see an option under feed settings to re-enable auto-play for these pieces as they're often my favorite and I don't want to miss them when scrolling through my feed.

**Comment by andrevenancio** [comment]
Good idea. adding this to the backlog. It might take a while to implement as we are prioritising more urgent features.
